migration.md

```diff
@@ -1,8 +1,10 @@
-React Swipeable v6 changes and migration
-=========================
+# React Swipeable v6 changes and migration
 
 ## Major Changes
+
 - removal of `<Swipeable>` component
   - _TODO_ add example for creating one and migrating
 
```

Comment:
💭 add new section for `New Features` and add this to that list, and remove from migrate section

Reply:
added to `New Features` section